### DISCUSSION TITLE
@Field with jest, any help is welcome

### DISCUSSION THREAD

**Discussion by kdttv** [MichalLytek/type-graphql#1621] [General]
I trying to test an class with decorator "InputType" but i wanna test the graphql schema... exist any way of me doing that?
Exemple

`@InputType
class A {
@Field((type) => [Number, Number], { nullable: true })
  arr2: [number, number];
};`
I'm trying validate if my decorator "Field" is an [Number, Number] with jest... idk if i could extract this information using something like buildSchema... idk...

**Comment by javatunix** [comment]
If i'm understand search something like "IntrospectionQuery" maybe it could help you.

**Comment by MichalLytek** [comment]
@kdttv You can't have tuples in graphql. `[Number, Number]` is invalid.
